app/views.py

```diff
@@ -1303,3 +1327,49 @@ def delete_item_from_folder(request, folder_id, item_id):
         res = delete_items_in_folder(item_id, folder_id, item_type)
         return Response(res + " Item Deleted in Folder", status.HTTP_202_ACCEPTED)
     return Response(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+def _paginate( dataset,page,limit):
+    start = (page - 1) * limit
+    end =  start + limit
+    return dataset[start:end]
+
+
+
+@api_view(["GET"])
+def dowell_centre_template(request, company_id):
+    """Fetch Dowell Knowledge centre templates."""
+    data_type = request.query_params.get("data_type", "Real_Data")
+    if not validate_id(company_id):
+        return Response("Something went wrong!", status=status.HTTP_400_BAD_REQUEST)
+    print("Valid ID!!")
+    templates = get_template_list(company_id, data_type)
+
+    page = int(request.GET.get('page', 1))
+    print(page)
+    templates = _paginate(templates,page,50)
+
+    return Response(
+        {"templates": templates},
+        status=status.HTTP_200_OK,
+    )
+
+
+@api_view(["GET"])
+def dowell_centre_documents(request, company_id):
+    """List of Created Documents."""
+    data_type = request.query_params.get("data_type", "Real_Data")
```

Comment:
Do not provide optional Real_data query param.
Check the existting code for how to do this